### ISSUE TITLE
Zoom to feature taken from nominatim results does not fit the viewer

### ISSUE THREAD

**Issue by MV88** [geosolutions-it/MapStore2#3932]
### Description
the features such as cities added from nominatim are not zoomed correctly after being selected

### In case of Bug (otherwise remove this paragraph)

*Browser Affected*
(use this site: https://www.whatsmybrowser.org/ for non expert users)
- [ ] Internet Explorer
- [ ] Chrome
- [ ] Firefox
- [ ] Safari

*Browser Version Affected*

- Indicate the browser version in which the issue has been found

*Steps to reproduce*

- search "liguria, genova"
- click on first result
![image](https://user-images.githubusercontent.com/11991428/60963862-4098b800-a312-11e9-8ff6-be961402999b.png)

*Expected Result*

![image](https://user-images.githubusercontent.com/11991428/60963925-602fe080-a312-11e9-8c4b-7bde86d370dd.png)

*Current Result*

- Describe here the current behavior
![image](https://user-images.githubusercontent.com/11991428/60963886-51492e00-a312-11e9-97a4-7b7b891b9541.png)
![image](https://user-images.githubusercontent.com/11991428/60963964-7178ed00-a312-11e9-91eb-3eec224f148c.png)

### Other useful information (optional):
this happen also on prod
this is a good issue for @offtherailz

**Comment by kasongoyo** [comment]
I can see that the fix for this issue requires a changing of the default max extent defined in map config. Changing of max extent is not the decision I can make alone so perhaps @offtherailz or @mbarto can have a look at this issue and help out to decide.

**Comment by francoperversi** [comment]
This bug, for features like cities, provinces, regions and states is fixed. However I notice that when you search a Continent, there is still a problem (see GIF)

![Continenti](https://user-images.githubusercontent.com/54808782/64411003-8fea3300-d08c-11e9-8601-cc9f517c0ed9.gif)
 
I ask to @tdipisa if this problem is related with this issue #3932

**Comment by tdipisa** [comment]
@francoperversi yes it is a normal behavior because in these cases Nominatim returns a point.
